lib/rules/disallow-spaces-inside-array-brackets.js

```diff
@@ -110,7 +110,7 @@ module.exports.prototype = {
                     token: openBracket,
                     nextToken: afterOpen,
                     spaces: 1,
```

Comment:
No, yeah it doesn't mean anything for `noWhitespaceBetween`, only for `whitespaceBetween`.